testing/xtrabackup/sigevent_musl.patch

```diff
@@ -0,0 +1,39 @@
+--- ./mysys/posix_timers.c.orig
++++ ./mysys/posix_timers.c
+@@ -22,6 +22,17 @@
+ #include <string.h>         /* memset */
+ #include <signal.h>
+
++struct linux_sigevent {
```

Comment:
This looks ugly hack. It probably also breaks horribly because timer_create() is libc call, and expects one to use libc definition of the struct sigevent.

The correct solution is likely to change in the same file something like:
```
-#ifndef sigev_notify_thread_id
+#if !defined(sigev_notify_thread_id) && defined(__GLIBC__)
 #define sigev_notify_thread_id   _sigev_un._tid
 #endif
```